### PR TITLE
docs: Show Application Session Activity example

### PR DESCRIPTION
**Pull request by stevenGravy**
Added how to retrieve Application Session activity.

### CODE REVIEW

**Comment by zmb3** [comment]
Feels like this is the wrong place for this info. These are step by step guides that walk you through a specific workflow. The app session content you're adding seems more like reference material, not tied to any particular guide.

**Comment by stevenGravy** [comment]
> Feels like this is the wrong place for this info. These are step by step guides that walk you through a specific workflow. The app session content you're adding seems more like reference material, not tied to any particular guide.

The getting started shows a grafana example so that includes the final step of getting the session activity.  Often we get questions on what app access records and how to access it.

Otherwise or in addition we could consider having a app access audit log guide to give what recording you would get for all the different types of app access activity.  There are other recordings you get like the AWS interaction.

File: docs/pages/includes/application-access/app-session.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Teleport records the HTTP activity within an Application Session. Only metadata
+such as path and status code from the HTTP activity is recorded, not the data
+sent or received by the application client.
+
+Within the Audit Log the `session_chunk_id` of the `App Session Data` record is
+the session id that activity is recorded against.  Once the `Session Uploaded`
+record is uploaded with that session id the activity is retrievable.
```
Comment (alexfornuto) [inline docs/pages/includes/application-access/app-session.mdx:7]: What about instances with local log storage? I don't get a `Session Uploaded` record, does that mean I can not get the ID and/or retrieve the logs with `tsh`?

**Comment by alexfornuto** [comment]
I think I agree with @zmb3 that this content should be a unique page.

**Comment by ptgott** [comment]
@stevenGravy Checking on this one. Are you still planning on moving it forward? Thanks!

**Comment by ptgott** [comment]
@stevenGravy Just checking to see what your plans are for this PR. Thanks!

**Comment by stevenGravy** [comment]
Closing to put in a separate page.

**Pull request closed without merging**